photon-api/src/main/scala/com/linkedin/photon/ml/util/MathUtils.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
```

Comment:
We can, it's easy enough. As a point of reference, I think the copyright is 95 years after publication? So we're safe until 2111.